lib/Swagger/src/Discussion/Models/LocalDateTime.php

```diff
@@ -0,0 +1,581 @@
+
+        return $this;
+    }
+
+    /**
+     * Gets day_of_month
+     * @return int
+     */
+    public function getDayOfMonth()
+    {
+        return $this->container['day_of_month'];
+    }
+
+    /**
+     * Sets day_of_month
+     * @param int $day_of_month
+     * @return $this
+     */
+    public function setDayOfMonth($day_of_month)
+    {
+        $this->container['day_of_month'] = $day_of_month;
+
+        return $this;
+    }
+
+    /**
+     * Gets year
+     * @return int
+     */
+    public function getYear()
+    {
+        return $this->container['year'];
+    }
+
+    /**
+     * Sets year
+     * @param int $year
+     * @return $this
+     */
+    public function setYear($year)
+    {
+        $this->container['year'] = $year;
+
+        return $this;
+    }
+
+    /**
+     * Gets month_value
+     * @return int
+     */
+    public function getMonthValue()
+    {
+        return $this->container['month_value'];
+    }
+
+    /**
+     * Sets month_value
+     * @param int $month_value
+     * @return $this
+     */
+    public function setMonthValue($month_value)
+    {
+        $this->container['month_value'] = $month_value;
+
+        return $this;
+    }
+
+    /**
+     * Gets chronology
+     * @return \Swagger\Client\Discussion\Models\Chronology
+     */
+    public function getChronology()
+    {
+        return $this->container['chronology'];
+    }
+
+    /**
+     * Sets chronology
+     * @param \Swagger\Client\Discussion\Models\Chronology $chronology
+     * @return $this
+     */
+    public function setChronology($chronology)
+    {
+        $this->container['chronology'] = $chronology;
+
+        return $this;
+    }
+    /**
+     * Returns true if offset exists. False otherwise.
+     * @param  integer $offset Offset
+     * @return boolean
+     */
+    public function offsetExists($offset)
+    {
+        return isset($this->container[$offset]);
+    }
+
+    /**
+     * Gets offset.
+     * @param  integer $offset Offset
+     * @return mixed
+     */
+    public function offsetGet($offset)
+    {
+        return isset($this->container[$offset]) ? $this->container[$offset] : null;
+    }
+
+    /**
+     * Sets value based on offset.
+     * @param  integer $offset Offset
+     * @param  mixed   $value  Value to be set
+     * @return void
+     */
+    public function offsetSet($offset, $value)
+    {
+        if (is_null($offset)) {
+            $this->container[] = $value;
+        } else {
+            $this->container[$offset] = $value;
+        }
+    }
+
+    /**
+     * Unsets offset.
+     * @param  integer $offset Offset
+     * @return void
+     */
+    public function offsetUnset($offset)
+    {
+        unset($this->container[$offset]);
+    }
+
+    /**
+     * Gets the string presentation of the object
+     * @return string
+     */
+    public function __toString()
+    {
+        if (defined('JSON_PRETTY_PRINT')) { // use JSON pretty print
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)